magefiles/magefile.go

```diff
@@ -360,6 +360,7 @@ func Dotenv() error {
 	myEnv["COMMONFATE_ACCESS_REMOTE_CONFIG_URL"] = cfg.Deployment.Parameters.ExperimentalRemoteConfigURL
 	myEnv["COMMONFATE_REMOTE_CONFIG_HEADERS"] = cfg.Deployment.Parameters.ExperimentalRemoteConfigHeaders
 	myEnv["COMMONFATE_GRANTER_V2_STATE_MACHINE_ARN"] = o.GranterV2StateMachineArn
+	myEnv["COMMONFATE_ENABLE_CRON_HEALTH_CHECK_IN_DEV"] = cfg.Deployment.Parameters.EnableCronHealthCheckInDev
```

Comment:
I don't believe that we need this environment variable as the configuration comes from the deployment.yml file

```suggestion
```